smoothrescale/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion '29.0.2'
 
     defaultConfig {
-        minSdkVersion 11
+        minSdkVersion 14
```

Comment:
@budowski do you think this will cause a problem? I don't think this is actually necessary since I ended up explicitly skipping the smoothrescale tests, but it doesn't seem like it will do any harm either.